### ISSUE TITLE
slice of n nodes from nodeset of n nodes doesn't work

### ISSUE THREAD

**Issue by brianjmurrell** [cea-hpc/clustershell#436]
Quite simply:

```bash
$ cluset -e -I 1 wolf-104vm10

$ cluset -e -I 1 wolf-104vm10,2
wolf-104vm10
$ cluset -e -I 2 wolf-104vm10,2

```

The slice of *all* from a nodelist should work, shouldn't it?  Having to special-case handle slicing all nodes seems wasteful.  To be sure, if I asked for `n+1` out of a list of `n`, that's a problem.  But asking for `n` out of `n` should _just work_, yes?

**Comment by brianjmurrell** [comment]
Hrm.  Re-reading about slicing... it seems I want to use `-I 0`.  But that gives me the last element of the 2 node list.

Maybe I should just ask: how do I get the first item in a nodeset?  I.e. for the set `1,2,3` how do I get `1`?  Ditto for the set `1`.

**Comment by brianjmurrell** [comment]
I think what is messing me up here is that it seems that `NodeSet` is sorted, not necessarily the order I provide to the command.  It's clear to see how this can yield results other than what one was expecting.  Is there any way to disable this sorting and treat the list as it was given to the `cluset` command?

**Comment by brianjmurrell** [comment]
Closed in favour of #437.
